frontend/src/layouts/NavigationBar/navigation-data.ts

```diff
@@ -6,7 +6,15 @@
 import { type NavigationData } from "./navigation-bar";
 
 // Text to display and the corresponding link
-export const unAuthenticatedNavItems: NavigationData = [
+
+export const baseNavItems: NavigationData = [
+  {
+    text: "Home",
+    link: "/"
```

Comment:
Current home route for "home" is `/tournaments` since we dont have anything else to show. There is a home route constant you can use.